algosdk/future/transaction.py

```diff
@@ -2393,7 +2393,7 @@ def verify(self, message):
                 try:
                     verify_key.verify(message, subsig.signature)
```

Comment:
@algochoi Opening a thread to clarify a point I missed earlier:

> I didn't change the v1 transaction methods, but the problem is there as well 

I'm unfamiliar with the _v1_ vs other version semantics.  As a sanity check:  Is there a reason _not_ to apply the PR's treatment to to _v1_ transaction methods?
* From looking around, I gathered you're referring to https://github.com/algorand/py-algorand-sdk/blob/develop/algosdk/transaction.py#L1208.
* I'm asking the question assuming there _is_ a reason not to make the update.  If easier to discuss live, let's do so.

Reply:
Bingo, that's the right v1 transaction you're referring to.

We have several issues to deprecate the v1 module in the Python SDK (no longer supported) but haven't done it yet:
* https://github.com/algorand/py-algorand-sdk/issues/252
* https://github.com/algorand/py-algorand-sdk/issues/286

Reply:
@algochoi Thanks for the pointers.  I took note of these issues on the deprecation + graduated levels of support front.

Since it's been the prevailing convention seems to be to avoid updates, then I'll follow the convention.  I confirmed the last change to `algosdk/transaction.py` is from 08/2021.  

Though I'm following the convention, I'm noting for completeness that it feels a little uncomfortable to ignore the API because it's _not_ deprecated.

Resolving.